app/test/main_test.dart

```diff
@@ -1,11 +1,11 @@
-import 'package:app/main.dart';
+import 'package:app/humane_transport_app.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
   testWidgets('HumaneTransportApp says hello world',
       (WidgetTester tester) async {
     await tester.pumpWidget(HumaneTransportApp());
-    final textFinder = find.text('Hello World');
+    final textFinder = find.text('This is Home Page');
```

Comment:
This is fair, it's a dumb test and we should start considering injection soon to reuse pages when we notice places for refactoring - then the test will not need to be changed for simple things like this

Reply:
In fact, I think we shouldn't bother with tests that check for text. I think that's too dumb unless we testing like, warning popups. No need to change this test still.

Reply:
Will do comprehensive flutter testing once I acquired more knowledge about it. But for now, as mentioned before, my main goal here is to explore bottomnavbar widget